tests/sharedtria/communicate_active_fe_indices_01.cc

```diff
@@ -40,10 +40,7 @@ template <int dim>
 void test()
 {
   parallel::shared::Triangulation<dim>
-  triangulation (MPI_COMM_WORLD,
-                 ::Triangulation<dim>::none,
-                 false,
-                 parallel::shared::Triangulation<dim>::partition_zorder);
+  triangulation (MPI_COMM_WORLD);
```

Comment:
I think we need to specify ``zorder`` here or not?

Reply:
This test has one output file for each of the three configurations we currently support to at least have one test that is run for all of them.

Reply:
Is this necessary? I don't like alternative outputs if we can avoid them, because this makes it annoying to update the output in the future if needed.

Reply:
These are not alternative outputs. For each configuration, there is exactly one output file.
The filenames are:
- communicate_active_fe_indices_01.with_trilinos=false.with_metis=false.mpirun=3.output               
- communicate_active_fe_indices_01.with_trilinos=false.with_metis=true.mpirun=3.output                
- communicate_active_fe_indices_01.with_trilinos=true.with_trilinos_with_zoltan=true.mpirun=3.output

This is of course not necessary, but having at least one test for `p::s::Triangulation` with a `Zoltan` partititoner is clearly useful. So why not reuse an existing one?

Reply:
I am fine with having those outputs. Alternative is to create 3 different tests one with `with_zoltan=on`, another with `with_zoltan=off,with_metis=on` and one with `with_zoltan=off,with_metis=off`. But IMO it's more cumbersome than having a single test with 3 outputs.

Reply:
Alright, that makes sense.